vllm_gaudi/extension/ops.py

```diff
@@ -895,12 +898,19 @@ def set_weight_block_size(self, block_size: Tuple[int, int] = (128, 128)):
         self.block_size = block_size
 
     def get_dequant_weight(self):
-        return dequant_block_fp8_weight_naive(
-            self.weight,
-            self.scale_inv_fp8,
-            block_size=self.block_size,
-            dtype=self.high_precision,
-        )
+        if self.quant_method in [FusedMoeWeightScaleSupported.GROUP.value, FusedMoeWeightScaleSupported.BLOCK.value]:
```

Comment:
```suggestion
        if self.quant_method in [FusedMoeWeightScaleSupported.BLOCK.value]:
```
From what I understand,  `FusedMoeWeightScaleSupported.GROUP` is used for group-level granularity, such as `1x64`, which cannot be handled by `dequant_block_fp8_weight_naive`.